scripts/hooks/pre-commit

```diff
@@ -4,9 +4,10 @@ STAGED_GO_FILES=$(git diff --cached --name-only -- '*.go')
 if [[ $STAGED_GO_FILES == "" ]]; then
     echo "no go files updated"
 else
-    echo "Formatting go files"
+    echo "Formatting and vetting code go files"
```

Comment:
Adding vetting to the checking process.  Note:  You need to do a make install-hooks to get this installed locally